deployment/helm/memory-qos/values.yaml

```diff
@@ -13,8 +13,12 @@ resources:
   memory: 100Mi
 
 nri:
-  patchRuntimeConfig: false
-  pluginIndex: 40
+  plugin:
+    index: 90
+  runtime:
+    patchConfig: false
+#    registrationTimeout: 5s
+#    requestTimeout: 4s
```

Comment:
```suggestion
#    config:
#       registrationTimeout: 5s
#       requestTimeout: 4s
```
I'm not sure which one should be the final you preferred. But I noticed that some places are missing `.config` level or the vise versa, some places have unnecessary `.config`